warehouse/query-core/src/main/java/datawave/query/util/FuzzyAttributeComparator.java

```diff
@@ -0,0 +1,91 @@
+package datawave.query.util;
+
+import com.google.common.collect.Sets;
+import datawave.query.attributes.Attribute;
+import datawave.query.attributes.Attributes;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class FuzzyAttributeComparator {
```

Comment:
I know I wrote the original ticket with the fuzzy wording, but let's change this class name. Fuzzy matching is a mathematical operation and in the end we're just doing a comparison of the backing data values. 

Rename to `AttributeComparator` or something similar. Add a class level comment that we're identifying attributes with equivalent data values across attribute types.

Reply:
Done and done.